<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>TransAtlasDB</title>
    <link rel="stylesheet" href="style.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
      <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-89779000-1', 'auto');
  ga('send', 'pageview');

</script>
  </head>
  <body>
    <div class="wrapper">
      <header>
        <a href="index.html"><h1>TransAtlasDB</h1></a>
        <p>An Informatics system for Transcriptome Analysis Data</p>
        <br>
        <ul class="icons">
          <a href="index.html"><strong><center>What is TransAtlasDB?</center></strong></a>
        </ul>
        <ul class="icons">
          <a href="usage.html"><strong><center>Usage & Downloads</center></strong></a>
        </ul>
        <ul class="icons">
          <a href="toolkit.html"><strong><center>Toolkit</center></strong></a>
        </ul>
        <ul class="icons">
          <a href="tutorial.html"><strong>Tutorial</strong></a>
        </ul>
        <ul class="icons">
          <a class="select" href="sample.html"><strong>Sample Files format</strong></a>
        </ul>
        <br>
        <p class="view"><a href="https://github.com/modupeore/TransAtlasDB">View the Project on GitHub <small></small></a></p>       
        <ul>
          <li><a href="download/transatlasdb.zip">Download <strong>ZIP File</strong></a></li>
          <li><a href="download/transatlasdb.tgz">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://drive.google.com/uc?id=1GFECcPLSjOuVq0RUNzYG0LcAf2_OsP1U&export=download" target="_blank">Download <strong>VMA File</strong></a></li>
        </ul>      
      </header>
      <br>
      <section>
        <h2>TransAtlasDB Sample Format</h2>
        <p>This section discusses TransAtlasDB input data format and output formats.</p>
        <hr>
        <h3>INPUT FORMAT</h3>
        <div class="pushit">
          <p>TransAtlasDB accepts input data from the different software required for differential expression analysis. <br>
          The current prototype accepts outputs from the Tuxedo Suite - TopHat and Cufflinks. </p>
          Thus, the following information is required to use the TransAtlasDB database application;
          <ol type= "a">
            <li><strong>Sample Information.</strong></li>
            <p>The Sample information, or samples metadata, is the reference point of the corresponding results from RNAseq data and therefore important for data archival and retrieval of the various transcriptome analysis. TransAtlasDB preferably accepts the sample information through the <a href="http://www.faang.org" target="_blank">FAANG</a> sample submission spreadsheet template to <a href="https://www.ebi.ac.uk/biosamples/" target="_blank">BioSamples</a>. Details are provided <a href="https://www.ebi.ac.uk/seqdb/confluence/display/FAANG/Submission+of+samples+to+BioSamples" target="_blank">here</a>.<br>
            The FAANG sample submission spreadsheet template provides a detailed questionnaire for each sample and hence our database system was modeled to accept the FAANG excel template. However, the required fields in the spreadsheet are the Animal and Specimen sheets; with the <font color=#333>Animal-‘Sample Name’</font>, <font color=#333>Animal-‘Organism’</font>, <font color=#333>Specimen-‘Sample Name’</font> and <font color=#333>Specimen-‘Organism Part’</font> column filled. Also, the <font color=#333>Specimen-‘Derived from’</font> should be the same as the <font color=#333>Animal-‘Sample Name’</font> column of each sample.</p>
            The database system also accepts a tab-delimited file with the minimum required columns shown below: 
<table>
  <thead>
    <tr>
      <th style="text-align: left">Header</th>
      <th style="text-align: left">Status</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: left">Sample name</td>
      <td style="text-align: left"><code class="highlighter-rouge">required</code></td>
      <td style="text-align: left">Sample identification number</td>
    </tr>
    <tr>
      <td style="text-align: left">Sample description</td>
      <td style="text-align: left"><code class="highlighter-rouge">optional</code></td>
      <td style="text-align: left">Description of sample</td>
    </tr>
    <tr>
      <td style="text-align: left">Derived from</td>
      <td style="text-align: left"><code class="highlighter-rouge">required</code></td>
      <td style="text-align: left">Animal identification number</td>
    </tr>
    <tr>
      <td style="text-align: left">Organism</td>
      <td style="text-align: left"><code class="highlighter-rouge">required</code></td>
      <td style="text-align: left">Species name</td>
    </tr>
    <tr>
      <td style="text-align: left">Organism part</td>
      <td style="text-align: left"><code class="highlighter-rouge">required</code></td>
      <td style="text-align: left">Name of tissue</td>
    </tr>
    <tr>
      <td style="text-align: left">First name</td>
      <td style="text-align: left"><code class="highlighter-rouge">optional</code></td>
      <td style="text-align: left">Person's first name</td>
    </tr>
    <tr>
      <td style="text-align: left">Middle initial</td>
      <td style="text-align: left"><code class="highlighter-rouge">optional</code></td>
      <td style="text-align: left">Person's middle initial</td>
    </tr>
    <tr>
      <td style="text-align: left">Last name</td>
      <td style="text-align: left"><code class="highlighter-rouge">optional</code></td>
      <td style="text-align: left">Person's last name</td>
    </tr>
    <tr>
      <td style="text-align: left">Organization</td>
      <td style="text-align: left"><code class="highlighter-rouge">optional</code></td>
      <td style="text-align: left">Institutional name</td>
    </tr>
  </tbody>
</table>
<p><strong>To download the FAANG BioSamples template click <a href="FAANG_template.xlsx" target="_blank" download="FAANG_template.xlsx">here</a>.</strong><br>
<strong>To download the tab-delimited template click <a href="samples-template.txt" target="_blank" download="template.txt">here</a>.</strong><br>
Otherwise, the sample information can be manually inserted using SQL insert statements.</p> </ol>
            <ol type= "a" start=2>
              <li><strong>Alignment Information.</strong></li>
              The Alignment information is comprised of the alignment files generated from either RNAseq read mappers; TopHat2 or HISAT2.
              Depending on the RNAseq read mapper used, the expected files are;<br>
              <ol type="i"><li>
              BAM file ( suffix: <i><font color="#d14">.bam</font></i> )</li>
              <font size="2pt" color=#333><i>NOTE</i>: HISAT2 default alignment SAM file must be converted to BAM.</font></ul>
              
              <li>Summary file ( i.e. <i><font color="#d14">align_summary.txt</font></i> )</li>
              <font size="2pt" color=#333><i>NOTE</i>: HISAT2 alignment summary, produced as stardard output at default, must be saved as 'align_summary.txt' file</font></ul>
              
              <li>Bed files ( i.e. <i><font color="#d14">deletions.bed</font></i> , <i><font color="#d14">insertions.bed</font></i> , <i><font color="#d14">junctions.bed</font></i> )</li>
              <font size="2pt" color=#333><i>NOTE</i>: The bed files are optional.</font></ul>
              </ol>
            </ol><p></p>
            <ol type= "a" start=3>
            <li><strong>Expression Information.</strong></li>
              The Expression information consists of the genes FPKM (Fragments Per Kilobase of transcript per million) file generated using Cufflinks or StringTie, and genomic features read counts using various feature summarization tools like HTSeq-count and featurecount. The required files are;<br>
              <ol type="i"><li>
             <b> StringTie</b></li>
              <ul class="tools"><li>Transcripts GTF ( suffix: <i><font color="#d14">.gtf</font></i> )</li>
              <font size="2pt" color=#333><i>NOTE</i>: The transcripts details should be saved with the suffix '.gtf'</font></ul>
              
              <li><b>Cufflinks </b></li>
              <ul class="tools"><li>Genes fpkm tracking table ( i.e. <i><font color="#d14">genes.fpkm_tracking</font></i> ), or </li>
              <li>Transcripts GTF ( i.e. <i><font color="#d14">transcripts.gtf</font></i> )</li></ul>
              
              <li><b>Feature summarization </b></li>
              <ul class="tools"><li>Read Counts file ( suffix: <i><font color="#d14">.counts</font></i> )</li>
              <font size="2pt" color=#333><i>NOTE</i>: The read counts should be saved with the suffix '.counts'</font></ul></ol>
            </ol><p></p>
            <ol type= "a" start=4>
            <li><strong>Variant Information.</strong></li>
              The Variant information includes the variant VCF file ( suffix: <i><font color="#d14">.vcf</font></i> ) from variant callers, such as GATK, SAMtools (BCFtools).<br>
              Optionally, the functional annotations of variants predicted by different bioinformatics tools can also be provided in a tab-delimited format. TransAtlasDB currently accepts functional annotations from two annotation software;
              <ul type="square"><li>Ensembl Variant Effect Predictor (VEP) : The output file should be named with suffix: <i><font color="#d14">.vep.txt</font></i>.</li>
              <li>ANNOVAR : The default output file with suffix: <i><font color="#d14">multianno.txt</font></i> will be imported. </li></ul>
          </ol>
          The input data should be stored in a single folder for each sample, shown below. The folder name must be the sample name as denoted in the metadata.
          <center><figure><A name="folder"><img src="images/sampledir.png"></a>
        <figcaption>Directory structure layout for each sample. The Folder Name must be the sample name as denoted in the metadata. Output files from the specified software for RNAseq analysis, TopHat and Cufflinks (A), or HISAT and StringTie (B) can be stored in the parent folder, otherwise, can be reorganized in subfolders listed. The solid-line boxes are required, while the optional files are represented in dashed-line boxes.</figcaption></figure></center>
        </div>
        <hr/>
        <h3>OUTPUT FORMAT</h3></a>
        <div class="pushit">
          <p>
            TransAtlasDB outputs user-defined queries as a tab-delimited table.
          </p>
          <p>
            This table is the default output format which is accepted by most text editors or statistics tools such as Microsoft Excel, R and JMP software. <br>
            Asides from the tab-delimited format for exporting results, the variant information can be generated as a VCF output using the <d>-vcf</d> flag.</p>
          <h4>Variant Call Format</h4>
          <p>Predicted functional annotations and sample metadata are added in the INFO field of the VCF file, using the key “CSQ” and “MTD” respectively. <br>
          Data fields are encoded separated by “|”; the order of fields is written in the VCF header. <br>
          VCFs produced by TransAtlasDB follow the standard VCF version 4 file format, and can be used for further downstream analysis or visualization using various variant viewers such as the University of California Santa Cruz (UCSC) Genome Browser, Integrative Genomics Viewer (IGV), and other programs that accept VCF files.
          </p>
          
          </div>
          <hr>
        <p>Please click the menu items to navigate through this repository. If you have questions, comments and bug reports, please <a href="mailto:amodupe@udel.edu">email me </a>directly. </p>
        <p>Thank you very much for your help and support!</p>
      </section>
      <br>
      <a class="back-to-top" style="display: inline;" href="#">Back to Top</a>
      <footer>
        <p>This project is maintained by <a href="https://github.com/modupeore" target=_blank>Modupeore</a></p>
      </footer>
    </div>
    <script src="/TransAtlasDB/assets/js/scale.fix.js"></script>
    <script src=”//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js”></script>
    <script>
      jQuery(document).ready(function() {
        var offset = 250;
        var duration = 300;
        jQuery(window).scroll(function() {
          if (jQuery(this).scrollTop() > offset) {
            jQuery(‘.back-to-top’).fadeIn(duration);
          } else {
            jQuery(‘.back-to-top’).fadeOut(duration);
          }
        });
 
        jQuery(‘.back-to-top’).click(function(event) {
          event.preventDefault();
          jQuery(‘html, body’).animate({scrollTop: 0}, duration);
          return false;
        }) 
      });
    </script>
    </div>
    <script src="/TransAtlasDB/assets/js/scale.fix.js"></script>
  </body>
</html>
