<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>TransAtlasDB</title>
    <link rel="stylesheet" href="style.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
    
      ga('create', 'UA-89779000-1', 'auto');
      ga('send', 'pageview');
    </script>
    <link rel=”stylesheet” id=”font-awesome-css” href=”//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css” type=”text/css” media=”screen”>
  </head>
  <body>
    <div class="wrapper">
      <header>
        <a href="index.html"><h1>TransAtlasDB</h1></a>
        <p>An Informatics system for Transcriptome Analysis Data</p>
        <br>
        <ul class="icons">
          <a href="index.html"><strong><center>What is TransAtlasDB?</center></strong></a>
        </ul>
        <ul class="icons">
          <a href="usage.html"><strong><center>Usage & Downloads</center></strong></a>
        </ul>
        <ul class="icons">
          <a href="toolkit.html"><strong><center>Toolkit</center></strong></a>
        </ul>
        <ul class="icons">
          <a class="select" href="tutorial.html"><strong>Tutorial</strong></a>
        </ul>
        <ul class="icons">
          <a href="sample.html"><strong>Sample Files format</strong></a>
        </ul>
        <br>
        <p class="view"><a href="https://github.com/modupeore/TransAtlasDB">View the Project on GitHub <small></small></a></p>  
        <nav>
       <ul><h1><font color="#8f4724" size="5px">Navigation:</font></h1></ul>
       <ul><h2><A href="#tadinstall">Installation</a></h2></ul>
       <ul><h2><A href="#tadimport">Data import</a></h2></ul>
       <ul><h2><A href="#tadexport"><h2>Data retrieval</a></h2></ul>
       <ul><h2><A href="#web"><h2>Web Interface</a></h2></ul>
      </nav><br>
        <ul>
          <li><a href="download/transatlasdb.zip">Download <strong>ZIP File</strong></a></li>
          <li><a href="download/transatlasdb.tgz">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://drive.google.com/uc?id=1GFECcPLSjOuVq0RUNzYG0LcAf2_OsP1U&export=download" target="_blank">Download <strong>VMA File</strong></a></li>
        </ul>       
      </header>
      <br>
      <section>
        <h2>TransAtlasDB Quick Start Guide</h2>
        <p>TransAtlasDB is a sophisticated bioinformatics system, which incorporates both the Relational Database and NoSQL Database system.</p>
        <p>The relational database, <a href="https://www.mysql.com" target=_blank><b color="red">MySQL</b></a>, enforces data assurance and integrity and the NoSQL database system, <a href="https://sdm.lbl.gov/fastbit/" target=_blank><b>FastBit</b></a>, ensures rapid performance for querying big data.<br>
        Hence, both database systems are required to be installed and added to the systems’ or users’ executable path.</p>        
        <hr>
	<h3>BEFORE YOU BEGIN</h3>
	<div class="pushit">
	  <p>First, we need to download appropriate databases and required Perl modules from the <a href="usage.html" target=_blank><b color="red">Usage & Downloads page</b></a>, before proceeding to the next section.</p>
          <p>Assume that we have downloaded TransAtlasDB toolkit and unpacked the TAR package using <d>tar -xzvf transatlasdb.tgz</d> or the ZIP package using <d>unzip transatlasdb.zip</d>.<br>
   You will see that the directory contains several Perl programs with <i><font color="#d14">.pl</font></i> suffix.<br>
	 <strong>IMPORTANT:</strong> Make sure the TransAtlasDB folder is in your desired permanent location before proceeding, preferably the <d>$HOME</d> directory.<br>
<strong>NOTE:</strong> If you already added the transatlasdb path into your system executable path, then typing <d>tad-import.pl</d>
                would be okay instead of typing <d>perl tad-import.pl</d>. </p>
	</div><hr>
        <A name="tadinstall"><h3>INSTALLATION</h3></a>
        <div class="pushit">
          <p>The TransAtlasDB database system and necessary components need to be installed to a local disk using <b><font color="#000"><i>INSTALL-tad.pL</i></font></b> and user must have root privileges to install.<br>
					<strong>IMPORTANT:</strong> TransAtlasDB installation requires superuser (root) privileges.</p>
          <p>Only the <d>–password</d> argument is required if the user has root or admin privileges to the MySQL server, else specify the username and password.
        </div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for installing TransAtlasDB using defaults and mysql password as 'mysql-password'.</span>
  sudo perl <span class="s">transatlasdb-location/</span><span class="k">INSTALL-tad.pL</span> <span class="o">-</span><span class="nb">password</span> <span class="s">mysql-password</span>
</code></pre></div>
          <div class="pushit">
            <p>Otherwise, arguments such as the mysql username <d>-username</d> is required and optionally the database name <d>-databasename</d>, else the default name <i><font color="#d14">transatlasdb</font></i> will be assigned.</p>
          </div>
          <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for installing TransAtlasDB specifying username as 'mysql-username' and databasename as 'mysql-databasename'.</span>
  sudo perl <span class="k">INSTALL-tad.pL</span> <span class="o">-</span><span class="nb">p</span> <span class="s">mysql-password</span> <span class="o">-</span><span class="nb">username</span> <span class="s">mysql-username</span> <span class="o">-</span><span class="nb">databasename</span> <span class="s">mysql-databasename</span>
</code></pre></div>
          <div class="pushit">
          <p>The NoSQL folder-name <d>-fastbitname</d> and system's path <d>-location</d> can be optionally specified, if not, a default folder <i><font color="#d14">transatlasfb</font></i> will be created in the working directory.</p></div>
          <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for installing TransAtlasDB specifying username as 'mysql-username' and databasename as 'mysql-databasename' with nosql folder 'nosqlfolder'.</span>
  sudo perl <span class="k">INSTALL-tad.pL</span> <span class="o">-</span><span class="nb">p</span> <span class="s">mysql-password</span>  <span class="o">-</span><span class="nb">u</span> <span class="s">mysql-username</span> <span class="o">-</span><span class="nb">d</span> <span class="s">mysql-databasename</span> <span class="o">-</span><span class="nb">fastbitname</span> <span class="s"><i>nosqlfolder</i></span>
</code></pre></div>
          <div class="pushit"><h4>Connecting to the database using <font color="#000"><i>connect-tad.pL</i></font></h4>
<p>The installational module needs to be carried out once per local disk to prevent database access conflict. However, if such conflict arises user settings can be viewed and/or corrected for using <b><i>connect-tad.pL</i></b>.</p></div>
<div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for viewing user installation details.</span>
  <span class="k">connect-tad.pL</span></code>
<code><span class="c1"># Syntax for setting existing user installation details. Parameters are the same as the INSTALL-tad.pL script.</span>
  <span class="k">connect-tad.pL</span> <span class="o">-</span><span class="nb">p</span> <span class="s">mysql-password</span>  <span class="o">-</span><span class="nb">u</span> <span class="s">mysql-username</span> <span class="o">-</span><span class="nb">d</span> <span class="s">mysql-databasename</span> <span class="o">-</span><span class="nb">fastbitname</span> <span class="s">nosqlfolder</span>
</code></pre></div><hr/>

        <A name="tadimport"><h3>DATA IMPORT</h3></a>
        <div class="pushit">
          <p>Transcriptome analysis data can be imported using <b><font color="#000"><i>tad-import.pl</i></font></b>.</p>
          <h4>Importing Samples Metadata</h4>
          <p>The samples information, commonly known as, samples metadata consists of the relevant details to uniquely identify each specimen used for RNAseq.<br>
          The samples metadata can be imported via <d>-metadata</d> folowed by the FAANG sample submission spreadsheet or the template tab-delimited file. Instructions on samples metadata requirements can be found <a href="sample.html">here</a>.</p>
        </div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for importing the <b>example</b> FAANG metadata provided.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">metadata</span> <span class="s">example/metadata/FAANG/FAANG_GGA_UD.xlsx </span><br>
</code><code><span class="c1"># The second option to import a tab-delimited file using the <b>example</b> provided.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">metadata</span> <span class="o">-</span><span class="nb">t</span> <span class="s">example/metadata/TEMPLATE/metadata_GGA_UD.txt</span>
</code></pre></div>
          <div class="pushit">
            <h4>Importing Transcriptome Analysis Data</h4>
            <p>Transcriptome analysis results can be inserted using <d>-data2db</d>. The transcriptome profiling data or variant analysis data can be imported together via <d>-all</d> or separately using <d>-gene</d> for expresion profiling results or <d>-variant</d> for variant analysis results.</p>
          </div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for importing all RNASeq expression and variant results in for sample 'GGA_UD_1004'.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">data2db</span> <span class="o">-</span><span class="nb">all</span> <span class="s">example/sample_sxt/GGA_UD_1004</span>
<span class="c1"># Syntax for importing RNASeq expression profiling information in for sample 'GGA_UD_1004'.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">data2db</span> <span class="o">-</span><span class="nb">gene</span> <span class="s">example/sample_sxt/GGA_UD_1004</span>
<span class="c1"># Syntax for importing RNASeq variant results in for sample 'GGA_UD_1004'.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">data2db</span> <span class="o">-</span><span class="nb">variant</span> <span class="s">example/sample_sxt/GGA_UD_1004</span>
</code></pre></div>
        
        <div class="pushit">
            <h4>Importing Variant Annotation information</h4>
            <p>The variant functional annotations predicted from either VEP or ANNOVAR can also be imported using additional flags <d>-vep</d> or <d>-annovar</d> respectively and must be in their default tab-delimited format. If VEP, the filename should end with <i><font color="#d14">.vep.txt</font></i>, or else if ANNOVAR, the file having suffix <i><font color="#d14">.multianno.txt</font></i> will be accepted.</p>
          </div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for importing RNASeq variant results with VEP annotations for sample 'GGA_UD_1004'.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">data2db</span> <span class="o">-</span><span class="nb">variant</span> <span class="o">-</span><span class="nb">vep</span> <span class="s">example/sample_sxt/GGA_UD_1014</span>
<span class="c1"># Syntax for importing RNASeq data with ANNOVAR annotations for sample 'GGA_UD_1004'.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">data2db</span> <span class="o">-</span><span class="nb">all</span> <span class="s">example/sample_sxt/GGA_UD_1004</span> <span class="o">-</span><span class="nb">annovar</span>
</code></pre></div>
        <div class="pushit"><p><strong>NOTE:</strong> The analysis data should be stored in a single folder for each sample and the folder-name must be the same <b><i>sample name</i></b> as represented in the samples metadata. A typical folder directory structure is shown <a href="sample.html#folder">here</a>.</p>

        <center><figure><a href="images/import.png" target="_blank"><img src="images/import.png"></a>
        <figcaption>Data import procedure for samples metadata (A) and RNAseq data (B) respectively.</figcaption></figure></center>
        <hr/>
        <A name="tadexport"><h3>DATA RETRIEVAL</h3></a>
        <div class="pushit">
          <p>Transcriptome analysis data previously stored can be retrieved using <b><font color="#000"><i>tad-export.pl</i></font></b>.<br>
					<strong>NOTE:</strong> Analysis data can also be retrieved using the <strong><a href="#web">web interface</a></strong>.</p>
          <p>The export module offers two methods of extracting data from the database; one by performing data manipulation language (DML) SQL statements using <d>-query</d> and the second method consisting of pre-defined queries of research interest using <d>-db2data</d>.</p>
          <h4>Performing DML Statements</h4>
          <p>Users can execute direct SQL statements via <d>-query</d> to the relational database and the query must be wrapped with single quotation.<br>
          For instance, executing <i><font color="#d14">'show tables'</font></i> will retrieve all the rows currently in the database, which can be stored as a tab-delimited file.</p>
        </div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax to view all the tables in the database.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">query</span> <span class="s">'show tables'</span>
</code><code><span class="c1"># Syntax to retrieve all the rows in the Sample table and store the results in a tab-delimited file 'output.txt'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">query</span> <span class="s">'select * from Sample'</span> <span class="o">-</span><span class="nb">output</span> <span class="s">output.txt</span>
</code></pre></div>
        
          <div class="pushit">
          <h4>Executing Pre-defined Queries of Research Interest</h4>
          <p>The pre-defined user statements can be accessed via <d>-db2data</d>.</p>
          This method entails four options, that are of research interest: 
          <ol>
            <li>Average expression values of specified genes organized by the different tissues.</li>
            <h5>Average expression values via <i><font color="black">--avgfpkm</font></i> flag</h5>
            View the average, maximum and minimum expression (FPKM) values of specified genes given species and/or tissue(s) of interest.
          </ol></div>
            <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax to view the expression values of genes 'MST' and 'GDF' for the 'Gallus gallus' specie in the database.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">avgfpkm</span> <span class="o">-</span><span class="nb">gene</span> <span class="s">'MST,GDF'</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span>
</code><code><span class="c1"># To export prior syntax as a tab-delimited file 'output.txt'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">avgfpkm</span> <span class="o">-</span><span class="nb">gene</span> <span class="s">'MST,GDF'</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code></pre></div>
            
          <div class="pushit">
            <ol start="2">
            <li>Gene expression profiles across the different samples of the same organism. Specific samples can be selected.</li>
            <h5>Gene expression profiles via <i><font color="black">--genexp</font></i> flag</h5>
            View expression values of individual genes per sample.
            This is done to easily compare expression profiles of the different samples.
            </ol></div>
          <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax to view all the genes of all samples in the database for 'Gallus gallus' organism.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">genexp</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span>
</code><code><span class="c1"># To export prior syntax as a tab-delimited file 'output.txt'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">genexp</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code><code><span class="c1"># Syntax to view expression profiles of genes 'OPTN' and 'GDF' for samples 'GGA_UD_1004'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">genexp</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">gene</span> <span class="s">'OPTN,GDF'</span> <span class="o">-</span><span class="nb">sample</span> <span class="s">'GGA_UD_1004'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code></pre></div>
          
          <div class="pushit">
            <ol start="3">
            <li>Variant distribution of all, or selected chromosomes for individual samples in the database.</li>
            <h5>Variant distribution of chromosomes via <i><font color="black">--chrvar</font></i> flag</h5>
            Summary counts of the different variant types per chromosome for each sample.
            </ol></div>
          <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax to view chromosome variant counts for all samples in the database for 'Gallus gallus' organism.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">chrvar</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span>
</code><code><span class="c1"># To export prior syntax as a tab-delimited file 'output.txt'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">chrvar</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code><code><span class="c1"># Syntax to view variant count fo expression profiles of chromosomes 'chr1, chr2, chr3' for all samples.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">chrvar</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">chromosome</span> <span class="s">'chr1,chr2,chr3'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code><code><span class="c1"># Syntax to view variant count fo expression profiles of chromosomes 'chr1, chr2, chr3' for sample 'GGA_UD_1004'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">chrvar</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">chromosome</span> <span class="s">'chr1,chr2,chr3'</span> <span class="o">-</span><span class="nb">sample</span> <span class="s">'GGA_UD_1004'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code></pre></div>
          
          <div class="pushit">
            <ol start="4">
            <li>Variants and predicted functional annotations found in the organism or selected genes or chromosomes.</li>
            <h5>Variants and functional annotations via <i><font color="black">--varanno</font></i> flag</h5>
            View gene-associated variants and chromosomal region-associated variants with respective annotation information.
            and annotation information.
            </ol></div>
          <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax to view associated variants with respective annotation information for 'OPTN' gene in 'Gallus gallus' organism.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">varanno</span> <span class="o">-</span><span class="nb">gene</span> <span class="s">'OPTN'</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span>
</code><code><span class="c1"># Syntax to view associated variants with respective annotation for chromosomes 'chr1,chr4,chr7' in 'Gallus gallus' organism.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">varanno</span> <span class="o">-</span><span class="nb">chromosome</span> <span class="s">'chr1, chr4, chr7'</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span>
</code><code><span class="c1"># Syntax to view associated variants with respective annotation for chr1:50000-900000 in 'Gallus gallus' organism.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">varanno</span> <span class="o">-</span><span class="nb">chromosome</span> <span class="s">'chr1'</span> <span class="o">-</span><span class="nb">region</span> <span class="s">50000-900000</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span>
</code><code><span class="c1"># To export prior syntax as a tab-delimited file 'output.txt'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">varanno</span> <span class="o">-</span><span class="nb">chromosome</span> <span class="s">'chr1'</span> <span class="o">-</span><span class="nb">region</span> <span class="s">50000-900000</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">o</span> <span class="s">output.txt</span>
</code><code><span class="c1"># To export prior syntax as a vcf file 'output.vcf'.</span>
  <span class="k">tad-export.pl</span> <span class="o">-</span><span class="nb">db2data</span> <span class="o">-</span><span class="nb">varanno</span> <span class="o">-</span><span class="nb">chromosome</span> <span class="s">'chr1'</span> <span class="o">-</span><span class="nb">region</span> <span class="s">50000-900000</span> <span class="o">-</span><span class="nb">species</span> <span class="s">'Gallus gallus'</span> <span class="o">-</span><span class="nb">o</span> <span class="o">-</span><span class="nb">vcf</span> <span class="s">output.vcf</span>
</code></pre></div>
          <div class="pushit">
        <center><figure><a target="_blank" href="images/export.png"><img src="images/export.png"></a>
        <figcaption>Data retrieval procedure.</figcaption></figure></center>
        
        <hr>
   

	 <h3>ADDITIONAL PROGRAMS</h3>
        <div class="pushit">
          <h4>Interacting with TransAtlasDB using <font color="#000"><i>tad-interact.pl</i></font></h4>
          <p>If uncertain on how to proceed with the export module, the interaction module provides an easy-to-use menu-driven interface.<br>
          The menu offers seven choices of exploratory research interest and provides a detailed description of what can be done from the module.</p></div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax to begin the interaction module.</span>
  <span class="k">tad-interact.pl</span></code></pre></div>
          <div class="pushit"><p><strong>NOTE:</strong> The interaction module only outputs a small subset of results and further instructions on how to export the complete results will be displayed.</p>
				<h4>Deleting Data</h4>
        <p>Sample results can only be imported once to ensure data integrity, nonetheless, previously imported data can be cautiously deleted using <d>-delete</d> followed by sample name.
        </div>
        <div class="language-perl highlighter-rouge"><pre class="highlight"><code><span class="c1"># Syntax for deleting sample 'GGA_UD_1004' from the database.</span>
  <span class="k">tad-import.pl</span> <span class="o">-</span><span class="nb">delete</span> <span class="s">GGA_UD_1004</span>
</code></pre></div>
					</div>
        <hr>
				
					 <A name="web"><h3>WEB ENVIRONMENT</h3></a>
        <div class="pushit">
          TransAtlasDB web portal can be viewed using PHP and Apache. The URL for the web portal is provided after installation and at default it is located at <a href="http://localhost/web" target="_blank">http://localhost/web</a>.<br>
	Biologist can benefit from using the web interface to interact with the analysis data previously stored using the <a href="#tadimport">data import module</a>.<br>
					The web interface is designed into five major sections:
					<h4>About</h4>
					The about page gives the summary statistics of the samples data imported in the database, and consist of four parts.
					<ul><li>Organisms:<br>This sections displays the organisms and total number of samples present in the database.</li></ul>
						<center><a target="_blank" href="images/about1.png"><img src="images/about1.png"></a></center>
					<ul><li>Samples:<br>This section displays the organisms and total number of samples for each tissue in the database.</li></ul>
						<center><a target="_blank" href="images/about2.png"><img src="images/about2.png"></a></center>
					<ul><li>Samples Processed:<br>This section displays the summary counts of the samples already archived in the database.</li></ul>
						<center><a target="_blank" href="images/about3.png"><img src="images/about3.png"></a></center>
					<ul><li>Database Content:<br>This section displays the summary count of the variants and genes present in the database.</li></ul>
						<center><a target="_blank" href="images/about4.png"><img src="images/about4.png"></a></center>
					<h4>Data Import</h4>
					The data import page provides two methods of importing the samples metadata. Storing the large data files such as the gene expression profiling and variants analysis results can only be done using the Perl toolkit as explained above.
					<ul><li>File Upload:<br>Sample metadata are uploaded from a sample file, either using the sample tab-delimited <a href="download/samples-template.txt" target="_blank" download="template.txt">template</a> or FAANG <a href="download/FAANG_template.xlsx" target="_blank" download="FAANG_template.xlsx">spreadsheet</a>.<br>
					<strong>NOTE:</strong> At this point, only sample metadata files can be uploaded via the web interface. RNAseq analyses data files should be imported through the <a href="#tadimport">command-line toolkit</a>.</li></ul>
						<center><figure><a target="_blank" href="images/import1.png"><img src="images/import1.png"></a><figcaption>Data import file upload page.</figcaption></figure></center>
					<ul><li>Manual Entry:<br>Samples metadata can also be inserted by manually entering the information.</li></ul>
						<center><figure><a target="_blank" href="images/import2.png"><img src="images/import2.png"></a><figcaption>Data import manual entry page.</figcaption></figure></center>
					<h4>Metadata</h4>
					The metadata page displays the samples stored in the database and an overview of each sample storage-status as well as the analysis summary where applicable. The samples can be exported as a tab-delimited file.
					<ul><li>Samples Metadata and storage status:<br>View samples metadata currently in the database based on search criteria.</li></ul>
						<center><figure><a target="_blank" href="images/metadata1.png"><img src="images/metadata1.png"></a><figcaption>Displays all samples in the database by clicking on 'GO'.</figcaption></figure></center>
					<ul><li>RNAseq results metadata:<br>View samples analysis metadata.</li></ul>
						<center><figure><a target="_blank" href="images/metadata2.png"><img src="images/metadata2.png"></a><figcaption>Displays all sample analysis metadata in the database by clicking on 'GO'.</figcaption></figure></center>
					<h4>Gene Expression</h4>
					The gene expression data can be viewed based on the individual sample-gene expression or average expression profiles of multiple genes across all samples and tissues. By specifying one or more genes by their gene symbols, a fuzzy search is performed based on the characters specified. The results can be saved in a tab-delimited file.
					<ul><li>Gene Expression summary across all samples.</li></ul>
						<center><figure><a target="_blank" href="images/gene1.png"><img src="images/gene1.png"></a><figcaption>Displays all 'OPTN' genes expression summary statistics for all <i>G. gallus</i> samples.</figcaption></figure></center>
					<ul><li>Gene Expression for each sample</li></ul>
						<center><figure><a target="_blank" href="images/gene2.png"><img src="images/gene2.png"></a><figcaption>Displays all 'OPTN' gene expression for each sample selected.</figcaption></figure></center>
					<h4>Variants</h4>
					Variants can be viewed through querying gene symbols or chromosomal regions.
					<ul><li>Variant distribution per chromosome and sample</li></ul>
						<center><figure><a target="_blank" href="images/variant1.png"><img src="images/variant1.png"></a><figcaption>Displays the total number of variants, SNPs and INDELs found in chromosomes (chr1, chr2, chr3) for sample 'GGA_UD_1014'.</figcaption></figure></center>
					<ul><li>Variants found in genomic regions of specified genes. One or more genes can be specified and a fuzzy search is performed for each gene queried. The results can be exported as a tab-delimited file.</li></ul>
						<center><figure><a target="_blank" href="images/variant2.png"><img src="images/variant2.png"></a><figcaption>Displays all the variants and annotations for the 'OPTN' gene in <i>G. gallus</i> organism.</figcaption></figure></center>
					<ul><li>Variants found based on chromosomal regions specified. These results can be exported as a tab-delimited file or a VCF file for downstream analysis or visualization.</li></ul>
						<center><figure><a target="_blank" href="images/variant3.png"><img src="images/variant3.png"></a><figcaption>Displays all the variants and annotations found in <i>G. gallus</i> along chr1:6540000-6580000.</figcaption></figure></center>
				</div>
        <hr>

        <p>Please click the menu items to navigate through this repository. If you have questions, comments and bug reports, please <a href="mailto:amodupe@udel.edu">email me </a>directly. </p>
        <p>Thank you very much for your help and support!</p>
      </section>
      <br>
      <!--https://getflywheel.com/layout/add-sticky-back-top-button-website/-->
      <a class="back-to-top" style="display: inline;" href="#">Back to Top</a>
      <footer>
        <p>This project is maintained by <a href="https://github.com/modupeore" target=_blank>Modupeore</a></p>
      </footer>
    </div>
    <script src="/TransAtlasDB/assets/js/scale.fix.js"></script>
    <script src=”//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js”></script>
    <script>
      jQuery(document).ready(function() {
        var offset = 250;
        var duration = 300;
        jQuery(window).scroll(function() {
          if (jQuery(this).scrollTop() > offset) {
            jQuery(‘.back-to-top’).fadeIn(duration);
          } else {
            jQuery(‘.back-to-top’).fadeOut(duration);
          }
        });
 
        jQuery(‘.back-to-top’).click(function(event) {
          event.preventDefault();
          jQuery(‘html, body’).animate({scrollTop: 0}, duration);
          return false;
        }) 
      });
    </script>
  </body>
</html>
